arangod/Cache/Manager.h

```diff
@@ -36,9 +36,10 @@
 #include "Cache/Transaction.h"
 #include "Cache/TransactionManager.h"
 
-#include <stdint.h>
+#include <array>
 #include <atomic>
 #include <chrono>
+#include <cstdint>
 #include <map>
 #include <memory>
 #include <stack>
```

Comment:
this line can be removed now, right?
```suggestion
```

Reply:
No, `std::stack` is still in use. I just replaced a C-Array of stacks with an `std::array` of stacks.

Reply:
ok